Source/Core/DolphinQt2/Config/Mapping/MappingWidget.cpp

```diff
@@ -31,6 +31,28 @@ MappingWindow* MappingWidget::GetParent() const
   return m_parent;
 }
 
+bool MappingWidget::IsIterativeInput() const
+{
+  return m_parent->IsIterativeInput();
+}
+
+void MappingWidget::NextButton(MappingButton* button)
+{
+  auto iterator = std::find(m_buttons.begin(), m_buttons.end(), button);
+
+  iterator++;
```

Comment:
This is undefined behaviour. You need to check whether or not the iterator is `m_buttons.end()` before incrementing it.